docs/HACKING.md

```diff
@@ -210,7 +210,25 @@ make git.merge~%                    Draft-merge another branch, eg `make git-mer
 make git.conflicts                  Find any git conflicts in ./forks
 ```
 
-## What happens when I get this error?
+## Troubleshooting
+
+### EACCES Permissions Error
+
+If you get a permissions error when following any of the steps, run the following command and it should be fixed:
+
+```shell
+cd ~/bonfire && sudo chown -R yourusername:yourusername .
```

Comment:
maybe saying to run the chown in their bonfire directory, since not everyone would have it at ~/bonfire

Reply:
I suppose it could be changed to `cd bonfire`, since that is what the HACKING.md documentation says

Reply:
I changed it in the new PR